api_reference.md

```diff
@@ -607,19 +622,26 @@ param = {
   }
 }
 
-charge_response = Midtrans.update_subscription("d137e7f4-9474-4fc2-9847-672e09cb16f6", param)
+response = Midtrans.update_subscription("d137e7f4-9474-4fc2-9847-672e09cb16f6", param)
 
-charge_response.body == {
+# this will be Hash representation of the API JSON response:
+puts response.data == {
         "status_message": "Subscription is updated."
 }
 ```
 
-#### `Midtrans::Result`
-
+## Get json field value from Midtrans API result
 ```ruby
 result = Midtrans.charge(...)
 
-result.class # => Midtrans::Result
+# If in the future there is a new attribute in the Midtrans Result.
+# Recommended retrieve like example below.
+# :new_attribute is the new field name in API response JSON
+result.data[:new_attribute]
+
+# Alternatively you can also access like below.
```

Comment:
In previous comment you mention `retrieve`, but here you mention `access`, choose one and be consistent with the wording. Consistency is important when writing documentation.